connectors/http/http-base/src/main/java/io/camunda/connector/http/base/model/HttpCommonRequest.java

```diff
@@ -43,7 +43,16 @@ public class HttpCommonRequest {
   @FEEL
   @NotBlank
   @Pattern(regexp = "^(=|(http://|https://|secrets|\\{\\{).*$)", message = "Must be a http(s) URL")
-  @TemplateProperty(group = "endpoint", label = "URL")
+  @TemplateProperty(
+      group = "endpoint",
+      label = "URL",
+      constraints =
+          @TemplateProperty.PropertyConstraints(
+              notEmpty = true,
+              secretPattern =
+                  @TemplateProperty.SecretPattern(
+                      value = "^(http://|https://|\\{\\{).*$",
+                      message = "Must be a http(s) URL")))
```

Comment:
Now we have two annotations?

Reply:
Asking because now the pattern is duplicated in the `@Pattern` one and the `@TemplateProperty.SecretPattern`. I thought the idea was to reduce the duplication? Just curious :)